tests/test_attribute.py

```diff
@@ -60,6 +60,18 @@ async def test_simple_handler_rw(mocker: MockerFixture):
     set_mock.assert_awaited_once_with(1)
 
 
+@pytest.mark.asyncio
+async def test_handler_initialise(mocker: MockerFixture):
+    handler = SimpleHandler()
+    handler_mock = mocker.patch.object(handler, "initialise")
+    attr = AttrR(Int(), handler=handler)
+
+    await attr.initialise(mocker.ANY)
+
+    # The handler initialise method should be called from the attribute
+    handler_mock.assert_called_once_with(1)
```

Comment:
Why `1`?

Reply:
That's a mistake, I was thinking about it being called once!